parser/hintparser.y

```diff
@@ -562,6 +562,22 @@ Value:
 	{
 		$$ = strconv.FormatUint($1, 10)
 	}
+|	'+' hintIntLit
+	{
+		$$ = strconv.FormatUint($2, 10)
+	}
+|	'-' hintIntLit
```

Comment:
How is this syntax used?

Reply:
For negative numeric values

Reply:
[`d140007` (#45893)](https://github.com/pingcap/tidb/pull/45893/commits/d140007d9264c1ba62804d64a542aa1162ecb399)